userapi/storage/postgres/accounts_table.go

```diff
@@ -116,6 +116,7 @@ func (s *accountsStatements) InsertAccount(
 	localpart string, serverName spec.ServerName,
 	hash, appserviceID string, accountType api.AccountType,
 ) (*api.Account, error) {
+	// TODO: can we replace "UnixNano() / 1M" with "UnixMilli()"?
```

Comment:
Yes we can! :)

As a followup PR, use `spec.AsTimestamp(time.Now())` which does the same, and update `AsTimestamp` in gomatrixserverlib to use `UnixMilli()`.

Reply:
created a pull requests for gomatrixserverlib https://github.com/matrix-org/gomatrixserverlib/pull/447